### PR TITLE
Community contributions 2022-10-27

### PR DESCRIPTION
**Pull request by Jimbo4350**
- Fix variable name type: https://github.com/input-output-hk/cardano-node/pull/3724
- Update `Update simple-scripts.md`: https://github.com/input-output-hk/cardano-node/pull/3648
- Also install libnuma-dev and llvm packages: https://github.com/input-output-hk/cardano-node/pull/2961

### CODE REVIEW

**Review by olgahryniuk — APPROVED** [review]

File: doc/getting-started/install.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ For Debian/Ubuntu, use the following instead:
 
 ```bash
 sudo apt-get update -y
-sudo apt-get install automake build-essential pkg-config libffi-dev libgmp-dev libssl-dev libtinfo-dev libsystemd-dev zlib1g-dev make g++ tmux git jq wget libncursesw5 libtool autoconf liblmdb-dev -y
+sudo apt-get install automake build-essential pkg-config libffi-dev libgmp-dev libssl-dev libtinfo-dev libsystemd-dev zlib1g-dev make g++ tmux git jq wget libncursesw5 libtool autoconf liblmdb-dev llvm libnuma-dev -y
```
Comment (newhoggy) [inline doc/getting-started/install.md:45]: What are these extra dependencies for?

File: doc/reference/simple-scripts.md [hunk 2]
```diff
@@ -634,7 +633,6 @@ above this means <= 3000:
 
 ```bash
 cardano-cli transaction build-raw \
-    --invalid-hereafter 1000 \
     --invalid-hereafter 3000\
```
Comment (newhoggy) [inline doc/reference/simple-scripts.md:636]: Can we add a space before backslash while we're here?

**Review by newhoggy — APPROVED** [review]

File: doc/getting-started/install.md (hunk 1 above)
Comment (newhoggy) [reply doc/getting-started/install.md:45]: Those two are not hard dependencies and should not be required for a vanilla GHC installation.  Users may have installed a special version of GHC or other build tool that imposes this dependency.  I think for now list them as optional dependencies that may be required depending on the user's installation.

**Pull request merged**